README.md

```diff
@@ -236,7 +236,9 @@ App behavior parameters
   - Default: `false`
 - `htmlValidator`: Params to send to [html-validator](https://github.com/zrrrzzt/html-validator#usage) (if `enableValidator` is set to true). When `suppressWarnings` is set to true validation warnings will be hidden and only errors will be shown.
   - Default:  `{validator: 'http://html5.validator.nu', format: 'text', suppressWarnings: true}`
-  - Can also be disabled via request by setting a 'Partial' header and by passing `_disableValidator` into the model.
+  - Can be disabled for individual requests by sending the request header Partial with the value set to true or by passing _disableValidator to the model and setting it to true.
+- `validatorExceptions`: Params to customize which request request header or model value can be used to disable the HTML validator.
```

Comment:
"Use this to customize the name of the request header or model value that is used to disable the HTML validator."